src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -795,29 +795,75 @@ void TableDockWidget::exportGroupsToSpreadsheet() {
     }
 }
 
-void TableDockWidget::exportGroupsToSpreadsheet_polly(QString fileName) {
-    
+void TableDockWidget::prepareDataForPolly(QString writable_temp_dir,QString export_format,QString user_filename) {
```

Comment:
@Ky5595 exportGroupstoSpreadSheet already exists in tabledockwidget. The code is almost the same. Lets refactor this before merging.